inc/namespace.php

```diff
@@ -548,3 +560,58 @@ function add_ec2_instance_data_to_xray( array $trace ) : array {
 
 	return $trace;
 }
+
+/**
+ * Return an AWS CloudFront Client instance
+ *
+ * @return \Aws\CloudFront\CloudFrontClient
+ */
+function get_cloudfront_client() {
```

Comment:
```suggestion
function get_cloudfront_client() : CloudFrontClient {
```
(will also need to `use` the class)